src/pages/Index.vue

```diff
@@ -2,18 +2,120 @@
   <Layout>
 
     <!-- Learn how to use images here: https://gridsome.org/docs/images -->
-    <h1>Welcome to the Galaxy Mentoring Network</h1>
+    
+    <header class="header">
+    </header>
+      <body>
+        <b-container class="index-container">
+        <b-row>
+          <b-col>
+        <h1>Mentorship...for you!</h1>
 
-    <p>Mentorship...for you!</p>
+        <p>An all in one program targeted to speed up your growth!</p>
+        <a href="https://forms.gle/482EoBYtJozWpYCf9" class="btn" role="button" target="_blank">Apply Now</a>
```

Comment:
Maybe good to have the button being the width of the page. What do you think?

Reply:
It looked too long...this size is bold enough(to draw attention) and concise enough to blend with the site and the hub generally